src/index.mjs

```diff
@@ -63,40 +43,87 @@ function isIgnored(node, filter) {
  * links for `document`. Can also work off a supplied
  * DOM element or static array of URLs.
  * @param {Object} options - Configuration options for quicklink
- * @param {Array} options.urls - Array of URLs to prefetch (override)
- * @param {Object} options.el - DOM element to prefetch in-viewport links of
- * @param {Boolean} options.priority - Attempt higher priority fetch (low or high)
- * @param {Array} options.origins - Allowed origins to prefetch (empty allows all)
- * @param {Array|RegExp|Function} options.ignores - Custom filter(s) that run after origin checks
- * @param {Number} options.timeout - Timeout after which prefetching will occur
- * @param {Function} options.timeoutFn - Custom timeout function
+ * @param {Object} [options.el] - DOM element to prefetch in-viewport links of
+ * @param {Boolean} [options.priority] - Attempt higher priority fetch (low or high)
+ * @param {Array} [options.origins] - Allowed origins to prefetch (empty allows all)
+ * @param {Array|RegExp|Function} [options.ignores] - Custom filter(s) that run after origin checks
+ * @param {Number} [options.timeout] - Timeout after which prefetching will occur
+ * @param {Number} [options.throttle] - The concurrency limit for prefetching
+ * @param {Number} [options.limit] - The total number of prefetches to allow
+ * @param {Function} [options.timeoutFn] - Custom timeout function
  */
-export default function (options) {
+export function listen(options) {
```

Comment:
Could you talk a little about the thought process of switching from a default to using `listen`? In practice, I think most users are likely to copy/paste the default code snippet we provide (and so I don't view this as a huge ergonomics change), but would be good to walk through reasoning :)

Reply:
Maybe there's not as much consensus as I think there is, but I think have mixed imports vs default-only and/or named-only is the trickiest of the three. So, for me it basically comes down to convention/simplicity. It's certainly not impossible to have mixed exports.

However, for UMD and CJS users, it does mean you have to call "listen" function via `quicklink.default()`